dials_data/definitions/quartz_processed.yml

```
@@ -0,0 +1,29 @@
+name: Data processing results for electron diffraction data recorded from quartz crystals
```

```suggestion
name: DIALS output of processing a quartz crystal electron diffraction data set
```
since it's specifically DIALS output we can keep this line consistent with other data sets, eg. mpro*processed